catalog/utilities/media_props_gen/db.py

```diff
@@ -0,0 +1,81 @@
+import re
+from dataclasses import dataclass
+from pathlib import Path
+from typing import Any, Literal
+
+
+LOCAL_POSTGRES_FOLDER = Path(__file__).parents[3] / "docker" / "upstream_db"
+SQL_PATH = {
+    "image": LOCAL_POSTGRES_FOLDER / "0003_openledger_image_schema.sql",
+    "audio": LOCAL_POSTGRES_FOLDER / "0006_openledger_audio_schema.sql",
+}
+sql_types = [
+    "integer",
+    "boolean",
+    "uuid",
+    "double precision",
+    "jsonb",
+    "timestamp with time zone",
+    "character varying",
+]
+sql_type_regex = re.compile(f"({'|'.join(sql_types)})")
+MediaType = Literal["audio", "image"]
```

Comment:
This is already present in `common/constants.py`:

https://github.com/WordPress/openverse/blob/002066fdea540ebba81c7574df526c7db285401d/catalog/dags/common/constants.py#L13

I know that this is able to run outside the docker container presently, but if we followed a similar pattern as the DAG documentation generation script, that would mean we could reuse _even more_ of the `just` recipe when combining the logic for those two! What do you think?